src/main/java/eu/tsystems/mms/tic/testerra/plugins/xray/synchronize/AbstractXrayResultsSynchronizer.java

```diff
@@ -352,16 +359,62 @@ public void onTestStatusUpdate(TestStatusUpdateEvent event) {
          */
         currentTestIssues.stream()
                 .peek(issue -> xrayMapper.updateTest(issue, methodContext))
-                .map(XrayTestExecutionImport.TestRun::new)
-                .peek(test -> updateTestImport(test, methodContext))
+                .map(issue -> {
+                    XrayTestExecutionImport.TestRun run = new XrayTestExecutionImport.TestRun(issue.getKey());
+                    this.updateTestInfoForImport(run, issue, methodContext);
+                    return run;
+                })
+                .peek(testRun -> updateTestRunForImport(testRun, methodContext))
                 .forEach(testRunSyncQueue::add);
 
         if (testRunSyncQueue.size() >= xrayConfig.getSyncFrequencyTests()) {
             flushSyncQueue();
         }
     }
 
-    private void updateTestImport(XrayTestExecutionImport.TestRun testRun, MethodContext methodContext) {
+    /**
+     * Update the Info object for creating or updating Xray tests.
+     * If no Info object is defined, the Xray test will not update.
+     */
+    private void updateTestInfoForImport(XrayTestExecutionImport.TestRun testRun, XrayTestIssue issue, MethodContext methodContext) {
+        if (this.getXrayMapper().shouldCreateNewTest(methodContext)) {
+
+            XrayTestExecutionImport.TestRun.Info info = new XrayTestExecutionImport.TestRun.Info();
+            info.setDescription(issue.getDescription());
+            info.setSummary(issue.getSummary());
+            info.setLabels(issue.getLabels());
+            info.setDefinition(issue.getSummary());
+            info.setType(TestType.AutomatedGeneric);
+            info.setProjectKey(issue.getProject().getKey());
+
+            /*
+             * The test's test type needs to be {@link TestType.Manual} to support test steps.
```

Comment:
Simple line comment instead of a three-line block containing one line of content?